clang/test/CIR/global_var_linkage.cpp

```diff
@@ -0,0 +1,11 @@
+// Linkage types of global variables
+// RUN: %clang_cc1 -std=c++20 -triple x86_64-unknown-linux-gnu -fclangir -emit-cir %s -o -  | FileCheck %s
+
+int aaaa;
```

Comment:
This new tests uses_this_convention and while the existing-uses-this, I suggest you-keep-this-uniform.